### PR TITLE
Add instructions for local upstash redis + few small fixes

### PR DESCRIPTION
**Pull request by sergey-tyan**

### CODE REVIEW

File: components/icons/ConfettiIcon.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const ConfettiIcon = ({ className }: Props) => {
       width={24}
       height={24}
       viewBox="0 0 24 24"
-      stroke-width={2}
+      strokeWidth={2}
```
Comment (sergey-tyan) [inline components/icons/ConfettiIcon.tsx:13]: to fix a small warning 
image

**Comment by sergey-tyan** [comment]
@dahal please take a look

File: .env.example [hunk 2]
```diff
@@ -16,8 +16,8 @@ SHADOW_DATABASE_URL="mysql://root:password@localhost:3306/envless_shadow"
 NEXTAUTH_SECRET="XXXXXXXXXXXXXXXXXX"
 NEXTAUTH_URL="http://localhost:3000"
 
-# Encryption
-ENCRYPTION_KEY="XXXXXXXXXXXXXXXXXX"
+# Encryption key must be 32 length
+ENCRYPTION_KEY="12345678123456781234567812345678"
```
Comment (dahal) [inline .env.example:20]: 👍🏼

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -87,10 +87,28 @@ SHADOW_DATABASE_URL="mysql://root:password@localhost:3306/envless_shadow"
...
+`go install github.com/mna/upstashdis/cmd/upstash-redis-rest-server@latest`
+
+Update your `.env` with
+```
+UPSTASH_REDIS_REST_URL="http://127.0.0.1:3010"
+UPSTASH_REDIS_REST_TOKEN="asd"
+```
+Start Local upstash redis server with
+
+`upstash-redis-rest-server --addr 127.0.0.1:3010 --redis-addr 127.0.0.1:6379 --api-token asd`
```
Comment (dahal) [inline CONTRIBUTING.md:111]: This is amazing, thanks @sergey-tyan

**Review by dahal — APPROVED** [review]

**Pull request merged**